lib/Dancer2/Core/App.pm

```diff
@@ -36,7 +36,7 @@ sub supported_engines { [ qw<logger serializer session template> ] }
 
 has _factory => (
     is      => 'ro',
-    isa     => Object['Dancer2::Core::Factory'],
```

Comment:
Type::Standard's Object type throws an exception when used like this since Object doesn't check isa.